react/features/filmstrip/components/native/Filmstrip.js

```diff
@@ -149,32 +180,87 @@ class Filmstrip extends Component<Props> {
     }
 
     /**
-     * Sorts a specific array of {@code Participant}s in display order.
+     * Returns the participant list in a format suitable for the
+     * {@code FlatList} component to display.
+     *
+     * @private
+     * @returns {Array<{participant: Object, key: string}>}
+     */
+    _getFilmstripData() {
+        const { _participants } = this.props;
+        const data = [];
+
+        // If _separateLocalThumbnail is false (e.g. on Android for now, see
+        // comment in constructor for more details) we render the local
+        // participant together with the remote participants, so we need to put
+        // it as the first element of the data array.
+        if (!this._separateLocalThumbnail) {
+            const { _localParticipant } = this.props;
+
+            data.push({
+                participant: _localParticipant,
+                key: _localParticipant.id
+            });
+        }
+
+        if (_participants) {
```

Comment:
you can get rid of this if by making sure you an array even if empty

Reply:
Sure, but then I can't gate the ```reverse()``` operation easily. This way I have an additional if, but I don't execute a reverse operation on something that doesn't need it

Reply:
When is _participants falsy?

Reply:
Good point, thanks! I changed it to ```_participants.length```